### ISSUE TITLE
SSTableLoader is using 9042 transport port even after overriding it with the "--port" option

### ISSUE THREAD

**Issue by a-vivona** [scylladb/scylla-tools-java#324]
**Context:**
I have created a ScyllaDB cluster at AWS.
Encryption of data in transit from client to node is enabled on port 9142.
Inter-node encryption is also enabled.
**The cluster does not allow unencrypted connections.**
From a connection outside my cluster, I can use CQLSH via TLS just fine. It works very well.

From outside, I try to use sstableloader with TLS.
I have specified port 9142, but I get the error message below.
**In the error message, I see that it is trying to connect to port 9042 and not 9142.**

```bash
sstableloader \
--nodes X.X.X.X --port 9142 --ssl --username ZZZ --password ZZZ \
-ks ~/projets/tls/cqlsh-keystore.jks -kspw ZZZ -ts ~/projets/tls/cqlsh-truststore.jks -tspw ZZZ \
~/projets/backup/snapshots/ter-sc1/app_1/users/
```

```
===== Using optimized driver!!! =====
WARN  15:17:46,042 Error creating netty channel to ec2-X-X-X-X.eu-west-3.compute.amazonaws.com/X.X.X.X:9042
com.datastax.shaded.netty.channel.ConnectTimeoutException: connection timed out: ec2-X-X-X-X.eu-west-3.compute.amazonaws.com/X.X.X.X:9042
        at com.datastax.shaded.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe$1.run(AbstractNioChannel.java:261)
        at com.datastax.shaded.netty.util.concurrent.PromiseTask.runTask(PromiseTask.java:98)
        at com.datastax.shaded.netty.util.concurrent.ScheduledFutureTask.run(ScheduledFutureTask.java:170)
        at com.datastax.shaded.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
        at com.datastax.shaded.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
        at com.datastax.shaded.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:503)
        at com.datastax.shaded.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
        at com.datastax.shaded.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
        at com.datastax.shaded.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:829)
com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: ec2-X-X-X-X.eu-west-3.compute.amazonaws.com/X.X.X.X:9042 (com.datastax.driver.core.exceptions.TransportException: [ec2-X-X-X-X.eu-west-3.compute.amazonaws.com/X.X.X.X:9042] Cannot connect))
        at com.datastax.driver.core.ControlConnection.reconnectInternal(ControlConnection.java:270)
        at com.datastax.driver.core.ControlConnection.connect(ControlConnection.java:109)
        at com.datastax.driver.core.Cluster$Manager.negotiateProtocolVersionAndConnect(Cluster.java:1921)
        at com.datastax.driver.core.Cluster$Manager.init(Cluster.java:1832)
        at com.datastax.driver.core.Cluster.init(Cluster.java:228)
        at com.datastax.driver.core.Cluster.connectAsync(Cluster.java:401)
        at com.datastax.driver.core.Cluster.connect(Cluster.java:352)
        at com.scylladb.tools.BulkLoader$CQLClient.<init>(BulkLoader.java:374)
        at com.scylladb.tools.BulkLoader.main(BulkLoader.java:1346)
```

I found a similar case on Cassandra:
https://stackoverflow.com/questions/70358944/sstabeloader-in-apache-cassandra-4-0-is-using-9042-transport-port-even-after-ove

However the proposed solution is not compatible with the version of sstableloader provided by Scylla, I can't specify the port together with the IP address with the -d option, it shows me an error message "Invalid IPv6 address".

**Comment by a-vivona** [comment]
By any chance, this is not coming from the BulkLoader.java file?
Maybe it's a mistake on my part, but I have the impression that the port option is well read but that it is overloaded just after in the code.

https://github.com/scylladb/scylla-tools-java/blob/1c4e1e7a7df633d1d57e07f762e181e6a14f88af/src/java/com/scylladb/tools/BulkLoader.java#L1067

https://github.com/scylladb/scylla-tools-java/blob/1c4e1e7a7df633d1d57e07f762e181e6a14f88af/src/java/com/scylladb/tools/BulkLoader.java#L1159

**Comment by a-vivona** [comment]
Fixed #325
